services/viva-ms/lambdas/syncWorkflow.js

```diff
@@ -0,0 +1,100 @@
+import to from 'await-to-js';
+import deepEqual from 'deep-equal';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import AWS from 'aws-sdk';
+const dynamoDbConverter = AWS.DynamoDB.Converter;
+
+import config from '../../../config';
+import params from '../../../libs/params';
+import * as dynamoDb from '../../../libs/dynamoDb';
+import * as request from '../../../libs/request';
+import hash from '../../../libs/helperHashEncode';
+
+const SSMParams = params.read(config.vada.envsKeyName);
+
+const CASE_WORKFLOWS_PATH = 'details.workflows';
```

Comment:
Kan med fördel läggas in i /libs/constants.js. Där försöker vi lägga in alla konstanter. Kanske inte relevant i detta läge då det inte är en environment-relaterad konstant.

Input?

Reply:
Inte env-relaterad men jag är helt öppen till förslag, här får ni bara förslå vad ni tycker är bäst. :)